dockstore-integration-testing/src/test/java/io/dockstore/webservice/WebhookIT.java

```diff
@@ -668,12 +668,52 @@ private void testDefaultVersion(WorkflowsApi client) {
         Workflow workflow2 = getFoobar2Workflow(client);
         assertNull(workflow2.getDefaultVersion());
         Workflow workflow = getFoobar1Workflow(client);
-        assertNull(workflow.getDefaultVersion());
+        assertEquals("master", workflow.getDefaultVersion());
```

Comment:
I think these tests work because you know that the "master" branch has been set as default on GitHub.
That's not entirely different from relying on the structure of workflow descriptors, but somehow I feel that this could use a code comment in case it changes/breaks in the future.